src/foam/u2/svg/graph/DAGView.js

```diff
@@ -151,12 +159,30 @@ foam.CLASS({
       g
         .callIf(! self.alreadyRendered_[node.id], function () {
           self.alreadyRendered_[node.id] = true;
-          this
-            .tag(self.nodeView, {
-              data: node.data,
-              position: coords,
-              size: Array(coords.length).fill(self.cellSize)
-            })
+
+          if ( self.selectedNodeId && self.nodeView.hasOwnAxiom("isSelected") ){
```

Comment:
Why does this if/else exist. Aren't both sets of code the same?

Reply:
oh my bad justs pushed

Reply:
Supposed to not have  isSelected in the else block

Reply:
Then make the args be var args = { ... } and then do if (... ) args.isSelect$ = ...; then just do .tag(self.nodeView, args);

Reply:
Done.